client/routing.js

```diff
@@ -320,12 +320,20 @@ const routeTxRequest = async (conf, txRequest) => {
     }
     execute(conf, txRequest)
       .then(result => {
+        const { web3 } = conf
         const { receipt, from } = result
+
         if (receipt && receipt.status == 1) {
```

Comment:
there should be a case for `else if (receipt && receipt.status == 0 || receipt.status == '0x00' || receipt.status == false) { .... }`